index.html

```diff
@@ -843,6 +843,64 @@ <h2>Skills</h2>
    
     </section>
     <!--about me section ends here -->
+     
+              
+    <!--contact me   -->
+    <section class="contact">
+
+      <div class="contact-container">
+
+          <div class="contact-container-desktop">
+           
+            <h2 id="contact">Contact me</h2>
+            
+            <p class="contact-P contact-P-desktop">
+              If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+              I’d love to help with it!
+            </p>
+
+              <!--  contact form section  -->
+              <form class="contact-form" action="https://formspree.io/f/xaygnnvv" method="POST">
+                
+                <ul class="form-section">
+
+                  <li>
+                    
+                  <input type="text" name="name" class="input-desktop" maxlength="30" placeholder="Enter full name"
```

Comment:
[Optional] 👍 Your input fields look good, but there are a couple of adjustments needed to match the design:

- [ ] [Optional] The border radius for each input field is currently set at 5px, whereas the design specifies it should be 8px. Please update your code to set the border-radius to 8px, ensuring it aligns with the design requirements.

- [ ] [Optional] Additionally, the design calls for a 1px border of a specified color for each input field. However, currently, this border is missing. To adhere to the design, please add the 1px border with the correct color to each input field.
![image](https://github.com/melkamu372/Portfolio-mobile-first/assets/55840999/0898f651-8207-4dc7-8aca-b2bf218a8425)
By making these changes, your input fields will be in line with the design, creating a cohesive and visually appealing UI. Keep up the great work, and I'm looking forward to seeing your UI match the design more closely! 🎨🚀